src/main.rs

```diff
@@ -0,0 +1,25 @@
+ extern crate clap;
+ use clap::{Arg, App};
+extern crate calamine;
+
+fn main() {
+    
+    let matches = App::new("Program2")
+    .about("Program to extract the rate index")
+    .arg(Arg::with_name("input_file")
+             .short("i")
+             .long("in")
+             .takes_value(true))
+    .arg(Arg::with_name("output_file")
+             .short("o")
+             .long("out")
+             .takes_value(true))
+    .get_matches();
+
+let input_file = matches.value_of("input_file").expect("provide a valid argument for input file TD_RateCard.xlsx");
+let output_file = matches.value_of("output_file").expect("provide a valid argument for output file 01-03-2020.txt");
```

Comment:
u are supposed to write proper error message this is a kind of info plus u r writing general error message here data doestn't make any sense,  u program will not run for a day only